react/util/utils.js

```diff
@@ -3,13 +3,19 @@ import { readFile as readFileFs } from "react-native-fs";
 import { Buffer } from "buffer";
 import ripe from "ripe-sdk";
 
-export const dateString = function (timestamp, { separator = "/", year = true } = {}) {
+export const dateString = function (
+    timestamp,
+    { separator = "/", year = true, reverse = false } = {}
+) {
     const date = new Date(timestamp * 1000);
     const day = date.getDate() < 10 ? `0${date.getDate()}` : date.getDate();
     let month = date.getMonth() + 1;
     month = month < 10 ? `0${month}` : month;
-    const yearText = year ? `${separator}${date.getFullYear()}` : "";
-    return day + separator + month + yearText;
+    const yearText =
+        year && !reverse ? `${separator}${date.getFullYear()}` : year ? date.getFullYear() : "";
+    return reverse
+        ? yearText + separator + month + separator + day
+        : day + separator + month + yearText;

```

Comment:
@BeeMargarida I believe this can be simplified:
- Build text as before (`const text = day + separator + month + yearText`)
- `return reverse ? text.split(separator).reverse().join(separator) : text`

Reply:
Still, it should go to ripe-commons, including to `dateStringUTC`

Reply:
Can it be done here and ripe-commons and then on another PR we remove these functions and use ripe-commons'?

Reply:
yep